### ISSUE TITLE
bootstrapCDN certificates are not valid

### ISSUE THREAD

**Issue by b264** [MaxCDN/bootstrap-cdn#790]
All https downloads are disabled due to 
```
The server presented a certificate that was not publicly disclosed using the Certificate Transparency policy. This is a requirement for some certificates, to ensure that they are trustworthy and protect against attackers.
```
in Chromium Version 53.0.2785.143

```
Your connection is not private

Attackers might be trying to steal your information from maxcdn.bootstrapcdn.com (for example, passwords, messages, or credit cards). NET::ERR_CERTIFICATE_TRANSPARENCY_REQUIRED
Subject: *.bootstrapcdn.com
Issuer: RapidSSL SHA256 CA
Expires on: Oct 13, 2017
Current date: Nov 14, 2016
```
from
`https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css`

**Comment by b264** [comment]
related https://knowledge.rapidssl.com/support/ssl-certificate-support/index?page=content&id=ALERT2165&actp=LIST&viewlocale=en_US

**Comment by jmervine** [comment]
@b264 Thanks for the report. @jdorfman is working with our CDN provider's support team on this, I believe.

Adding the twitter thread here for tracking as well: https://twitter.com/monobotBlog/status/798097238140387328


@jdorfman LMK if you need anything from me.

Cheers!

**Comment by b264** [comment]
https://sslmate.com/blog/post/ct_redaction_in_chrome_53

**Comment by b264** [comment]
I don't believe this is a bootstrap-cdn issue any more, but I'm not 100% certain

**Comment by jmervine** [comment]
@jdorfman Just FYI -- not debating the issue -- SSL tests appear to be okay.

https://www.ssllabs.com/ssltest/analyze.html?d=maxcdn.bootstrapcdn.com

**Comment by XhmikosR** [comment]
The current cert seems fine to me. I don't get how @b264 sees another certificate?

**Comment by EdgeCaseBerg** [comment]
I'm seeing this issue as well. 
```
Google Chrome	53.0.2785.101 (Official Build) (64-bit)
Revision	d68319683072a27031ebac6ac151e59f4190cab7-refs/branch-heads/2785@{#838}
OS	Mac OS X 
```
![screen shot 2016-11-15 at 10 26 46 am](https://cloud.githubusercontent.com/assets/1691564/20311512/11a9fe26-ab1e-11e6-9d4e-58a706cca3ba.png)

**Comment by jdorfman** [comment]
Troubling, I am not getting alerts when issues are created =/

@b264 @EdgeCaseBerg as @jmervine said I have been working with MaxCDN support and we have yet to find anything. Not saying that your issues are invalid, it is just taking longer to diagnose.

Are you still seeing the same issues?

**Comment by jdorfman** [comment]
@b264 @EdgeCaseBerg we believe it **might** have something to do with this: 
https://sslmate.com/blog/post/ct_redaction_in_chrome_53 (we use GeoTrust)

cc: @jmervine

**Comment by XhmikosR** [comment]
Isn't there a newer Chrome version out already? I'm a Firefox user, but I tried with the latest stable Chrome on my Windows 7 VM and works fine.

**Comment by kek91** [comment]
I get the same error message here ( `NET::ERR_CERTIFICATE_TRANSPARENCY_REQUIRED` ) when going to https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css

Sertificate details:

```
SSL Server Certificate
Issued To

Common Name (CN)	*.bootstrapcdn.com
Organization (O)	<Not Part Of Certificate>
Organizational Unit (OU)	<Not Part Of Certificate>
Issued By

Common Name (CN)	RapidSSL SHA256 CA
Organization (O)	GeoTrust Inc.
Organizational Unit (OU)	<Not Part Of Certificate>
Validity Period

Issued On	Thursday, October 13, 2016 at 2:00:00 AM
Expires On	Saturday, October 14, 2017 at 1:59:59 AM
Fingerprints

SHA-256 Fingerprint	DC 56 27 A2 52 C2 52 D6 D6 F6 80 D5 20 01 CC F4
39 F2 FF 26 33 C9 F7 66 D6 D5 AB 6E 85 EE 64 33
SHA-1 Fingerprint	7C 00 F2 22 04 A9 91 14 C3 05 89 D5 D7 3A 78 7C
BA BD A7 A5
```

**Comment by jdorfman** [comment]
@kek91 what are your Browser and OS?

**Comment by kek91** [comment]
@jdorfman Sorry, my post has been updated.

Browser: Chromium, v 53.0.2785
OS: Linux elementary 0.4 (Loki) 64-bit

**Comment by jdorfman** [comment]
@kek91 can you try updating to the latest version of Chrome (`54.0.2840.59`) and tell me what happens?

**Comment by jdorfman** [comment]
@kek91 what datascenter are you hitting http://debug-17.netdna-cdn.com/ ?

**Comment by kek91** [comment]
This is currently the latest Chromium version for my ppa and I currently don't have time to troubleshoot when something inevitably breaks my Linux, so I'll just get a vm up tomorrow and replicate the issue there. I'll keep you posted :)

Regarding your newest comment, I'm hitting the **MaxCDN London2 Datacenter**

**Comment by jdorfman** [comment]
@kek91 sounds good thanks so much

**Comment by jmervine** [comment]
@b264 && @EdgeCaseBerg -- Mind checking your datacenters as well... http://debug-17.netdna-cdn.com/

**Comment by kek91** [comment]
FYI, I just got home and there's no sertificate issues here (**StackPath AMS1 datacenter**).

**Comment by jdorfman** [comment]
@kek91 same system (OS & Browser)?

**Comment by kek91** [comment]
Oh sorry I was so focused on the physical location I forgot about that. Running Chrome, but it's updated to v 54.0.2840.87 so the issue could be resolved there as well. Windows 7 64-bit.

**Comment by jdorfman** [comment]
@kek91 all good just wanted to make sure. Yeah unfortunately this Chrome bug really screwed things up, but don't worry all of Google's properties are fine... 😠

**Comment by jdorfman** [comment]
I am going to close this for now. If anyone is having any further issues please open a new issue. 

Solution: Upgrade Chrome to 54 and the problem will be fixed.

**More details:**
Due to a [bug in Chrome](https://bugs.chromium.org/p/chromium/issues/detail?id=664177), versions of Chrome 53 that are more than 10 weeks old now display this error message for all websites using Symantec certificates that were issued on or after June 1, 2016 (including from Symantec-owned brands like Thawte and GeoTrust). As of Friday, November 11, 2016, **Chrome users can simply restart Chrome to avoid the bug. Chromium users must upgrade.**

**Comment by b264** [comment]
I personally did not see the problem resolve itself by restarting, however it does appear to be a bug in Chromium (and, by extension, Chrome) and not bootstrap-cdn.  I've also found a bunch more ssl sites with the same "issue".

TL;DR: use Firefox.

**Comment by jdorfman** [comment]
@b264 yes this year alone we have had to change the cert 2 times due to the unstable state of the digital SSL/TLS certificate infrastructure. As big as our service is, we refuse to change it again. The industry really needs to get it together since it is hurting not only service providers, but hundreds-of millions of people that rely on those services.

**Comment by EdgeCaseBerg** [comment]
@jmervine 
![screen shot 2016-11-17 at 10 35 17 am](https://cloud.githubusercontent.com/assets/1691564/20395639/a0f22830-acb1-11e6-97f7-e6eeea8d5d58.png)

**Comment by jdorfman** [comment]
@EdgeCaseBerg 

> It's affecting a bunch of other things too (AWS for example)

Can you please elaborate?

**Comment by b264** [comment]
@EdgeCaseBerg I also had to use Firefox for AWS and countless other things....  Chromium and Chrome are currently completely broken....

**Comment by jdorfman** [comment]
@b264 AWS had issues as well?

**Comment by b264** [comment]
It still does; it's unuseable

**Comment by jdorfman** [comment]
OMG this is unbelievable. I am working with our support to update the cert (again).

**Comment by b264** [comment]
![screenshot from 2016-11-17 10 57 08](https://cloud.githubusercontent.com/assets/7057665/20398948/df80c7ac-acb4-11e6-9e13-af4d27905c7a.png)

**Comment by jdorfman** [comment]
It makes me feel a little better that they are having issues as well. I am just appalled of how many careless mistakes have been made by Browser and SSL vendors this year.

**Comment by b264** [comment]
This seems to have just been fixed, in Chromium anyway

**Comment by suchakra** [comment]
we are still seeing it...

**Comment by suchakra** [comment]
net::ERR_INSECURE_RESPONSE suggests the SSL cert expired. Pointing to a different CDN gets the UI working again.

**Comment by petarpetrovic** [comment]
Hello @suchakra, are you able to replicate the issue in all web browsers on your end or just in Chrome? Also, could you please perform a traceroute towards the maxcdn.bootstrapcdn.com URL and paste it here in the comments? I'd like to get this checked out in more detail. Thanks!

**Comment by suchakra** [comment]
Cannot reproduce it anymore; Appears fixed now. It was on chrome.

**Comment by petarpetrovic** [comment]
@suchakra Thank you for confirming! If the issue reappears, please let us know here in the comments (the exact web browser and its version, along with the traceroute towards the subdomain would help us a lot along the way).

**Comment by sanjaygovindan** [comment]
@petarpetrovic  - Our corporate proxy is picking up the following:
https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css?ver=3.3.7

![image](https://user-images.githubusercontent.com/19772737/41280271-e21acc1c-6e71-11e8-9ddd-7a58c0845277.png)

**Comment by XhmikosR** [comment]
@sanjaygovindan: you are commenting on a pretty old issue. Also, personally I can't reproduce so make sure you need every info you can (including browser version obviously)

**Comment by sanjaygovindan** [comment]
@XhmikosR  - Thanks for the pointers. TBH, it's most likely proxy/filter related on our end. Have asked for the site to be whitelisted. Will see if that works.

**Comment by uaoleg** [comment]
I'm getting the following error

`NET::ERR_CERT_AUTHORITY_INVALID
Subject: *.bootstrapcdn.com

Issuer: Sectigo RSA Domain Validation Secure Server CA

Expires on: 14 oct 2020

Current date: 13 may 2020`

**Comment by XhmikosR** [comment]
@XhmikosR please make a new issue with the info filled. Everything seems to work fine here.
